man/systemd-analyze.xml

```diff
@@ -744,6 +744,20 @@ Service b@0.service not loaded, b.socket cannot be started.
         generators enabled will generally result in some warnings.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--return-error-on=<replaceable>MODE</replaceable></option></term>
+
+        <listitem><para>Controls verification of units and its dependencies and whether
+        <command>systemd-analyze verify</command> exits with an error or not. There are four possible
+        modes that can be passed to this option and these are either <command>none</command>, which is
```

Comment:
This is way too verbose. Please try to match surrounding style.

`There are four possible modes that can be passed to this option and these are either <command>none</command>, which is the default mode and never returns errors when warnings arise during verification of units, <command>warning-in-selected</command>, which …` →
`With <command>none</command>, don't return an error when warnings arise during verification of units. This is the default. With <command>warning-in-selected</command>, …` and so on.